site/src/components/help.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Help
+description: Help component in the Atlas Design System
+template: standard
+---
+
+# Help
+
+Help component can be paired with form's elements and contain additional information such as an error message, hints, etc.
+
+## Usage
+
+Here is an example of a help component textarea usage paired with an input and a label.
+
+```html
+<label class="label margin-bottom-xxs" for="input-demo">Label</label>
+<input class="input" id="input-demo" type="text" placeholder="Placeholder" />
+<p class="help margin-top-xxs">Help message</p>
+```
+
+### Validation states
+
+`help-error`/`help-warning`/`help-success` helps with visual reflection of validation states.
+
+| State   | Class                 | Preview                                                          |
+| ------- | --------------------- | ---------------------------------------------------------------- |
+| Error   | `.help .help-error`   | <p class="help help-error margin-top-none">Error message</p>     |
```

Comment:
+1

Reply:
Updated. Also updated the state from "Error" to "Danger"